Makefile

```diff
@@ -18,6 +18,7 @@ clean:
 	rm -rf build
 
 publish:
+	make build
```

Comment:
you can replace it by just making line above: `publish: build`

This is how you create dependencies in Makefile